bee-node/src/config.rs

```diff
@@ -29,7 +41,18 @@ pub(crate) struct NodeConfigBuilder {
 }
 
 impl NodeConfigBuilder {
-    pub(crate) fn finish(self) -> NodeConfig {
+    /// Creates a node config builder from the local config file.
```

Comment:
```suggestion
    /// Creates a node config builder from a local config file.
```

Reply:
no, it's **the**, because `from_file` has no arguments. We can think about making this more generic and give it a path.

Reply:
okay...I changed the method to `from_file<P: AsRef<Path>>(path: P)`, so that we can give it **a** file. :smile:

Reply:
YeahI was just thinking that we would have added the param anyway and definitely forget to change the doc.